### DISCUSSION TITLE
10分鐘看懂什麼是神經網路與其運作方式

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#406] [General]
https://www.ibm.com/cloud/learn/neural-networks
最近看到IBM介紹神經網路的運作方式，由於我從來沒有接觸這個領域，但看完後覺得還蠻好懂的，分享給大家。
透過一個淺顯易懂的例子：是否去衝浪？
來展示神經網路的運作原理，透過兩個變數相乘的總和來幫助人們做出決定。
∑ＷiＸi + bias = Ｗ1Ｘ1 + Ｗ2Ｘ2 + Ｗ3Ｘ3 + bias
這兩個條件中，其中一個是變數Ｘ，他只有可能是1或0，而另一個變數則是Ｗ則是條件對你而言的重要程度。

**Comment by 11timyyy** [comment]
連結好像有誤喔!

**Comment by chupochun** [comment]
https://www.ibm.com/tw-zh/cloud/learn/neural-networks

**Reply by chupochun** [reply]
這個連結應該可以使用

**Reply by liangyinglly** [reply]
沒綽 就是這個連結 謝謝你幫我更正！
我的文也改好連結了！

**Comment by sean1325418** [comment]
神經網路簡單說就是靠著電腦算出加權來決定哪些東西可以被當成是特徵，不過現在內部那些層機會都不知道真實是怎麼運行的，很多時候都是靠運氣來設置一些變數，如果有數學公式可以來揭開那些隱藏層在做甚麼那機器學習的進步應該會更快

**Comment by pekoranoinu** [comment]
原來神經網路是這麼一回事啊!

**Comment by Lynn12378** [comment]
非常容易理解的解釋! 講到AI就常常聽到神經網路，但是我都不太清楚是什麼意思，感謝樓主分享!

**Comment by ThomasMeng0515** [comment]
神經網路對許多人來說都是一個艱深難懂的原理，感謝分享容易理解的內容!

**Comment by PoooYen** [comment]
非常淺白的形容，感謝好文分享。

**Comment by Sazmafuji** [comment]
感謝分享，你的形容很好懂，而且有學到

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，讓我知道原來神經網路是什麼東西。

**Comment by HaoFish123** [comment]
感謝分享，大概聊解了甚麼是神經網路。

**Comment by xkllkx** [comment]
感謝分享，對神經網路有了基礎的認識。

**Comment by jason79461385** [comment]
上學期上課的時候看到這東西都不太知道是怎麼樣做的呢，看完文章也更了解了。

**Comment by Lai27330** [comment]
謝謝分享，感覺對神經網路有更多理解了。

**Comment by sys511613** [comment]
也很推李宏義教授的機器學習課程，個人覺得講得蠻淺顯易懂的，有興趣的可以看看
https://www.youtube.com/watch?v=7XZR0-4uS5s&list=PLJV_el3uVTsPM2mM-OQzJXziCGJa8nJL8

**Comment by minniewei** [comment]
感謝分享！！！我覺得在未來會非常有用！！

**Comment by treeann** [comment]
神經網路真的超難感謝分享

**Comment by jcpan821** [comment]
感謝分享~~

**Comment by stan890425** [comment]
感謝分享!!

**Comment by alexthebigpp** [comment]
感謝分享，獲益良多!

**Comment by etin2468** [comment]
感謝分享，對神經網路有更多理解了。

**Comment by Hchejxnc** [comment]
之前上課上到這一部分的時候，感覺還不是很了解神經網路是什麼，現在終於有些明白了!
